client/landing/stepper/declarative-flow/flows/site-migration-flow/paths/index.ts

```diff
@@ -0,0 +1,185 @@
+	queryParams: {
+		from?: string | null;
+		skipMigration?: boolean;
+	};
+} >( STEPS.PROCESSING.slug );
+
+export const credentialsPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+		authorizationUrl?: string;
+		backTo?: string;
+	};
+} >( STEPS.SITE_MIGRATION_CREDENTIALS.slug );
+
+export const upgradePlanPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+		destination?: string;
+		how?: string;
+	};
+} >( STEPS.SITE_MIGRATION_UPGRADE_PLAN.slug );
+
+export const alreadyWpcomPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+	};
+} >( STEPS.SITE_MIGRATION_ALREADY_WPCOM.slug );
+
+export const instructionsPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+	};
+} >( STEPS.SITE_MIGRATION_INSTRUCTIONS.slug );
+
+export const otherPlatformDetectedImportPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+		platform?: ImporterPlatform;
+	};
+} >( STEPS.SITE_MIGRATION_OTHER_PLATFORM_DETECTED_IMPORT.slug );
+
+export const applicationPasswordAuthorizationPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+		authorizationUrl?: string | null;
+		success?: string | null;
+		password?: string | null;
+		user_login?: string | null;
+	};
+} >( STEPS.SITE_MIGRATION_APPLICATION_PASSWORD_AUTHORIZATION.slug );
+
+export const fallbackCredentialsPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+	};
+} >( STEPS.SITE_MIGRATION_FALLBACK_CREDENTIALS.slug );
+
+export const supportInstructionsPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug?: string;
+		from?: string | null;
+		preventTicketCreation?: boolean;
+	};
+} >( STEPS.SITE_MIGRATION_SUPPORT_INSTRUCTIONS.slug );
+
+export const identifyPath = buildPathHelper< {
+	queryParams: { from: string | null };
+} >( STEPS.SITE_MIGRATION_IDENTIFY.slug );
+
+export const siteSetupImportListPath = buildPathHelper< {
+	queryParams: {
+		from: string | null;
+		siteId?: number | string;
+		siteSlug: string;
+		origin: string;
+		backToFlow: string;
+	};
+} >( `/setup/site-setup/${ STEPS.IMPORT_LIST.slug }` );
+
+export const calypsoImporterPath = buildPathHelper< {
+	queryParams: { engine: string; ref: string };
+	params: { siteSlug: string };
+} >( '/import/:siteSlug' );
+
+export const siteSetupImportWordpressPath = buildPathHelper< {
+	queryParams: {
+		siteId?: number | string;
+		siteSlug: string;
+		from: string;
+		option: 'content';
+		backToFlow: string;
+	};
+} >( '/setup/site-setup/importerWordpress' );
+
+export const calypsoOverviewPath = buildPathHelper< {
+	queryParams: { ref: string };
```

Comment:
why this path doesn't have a 
```
param: { siteSlug: string }
```
?

Reply:
Fixed here [47721ac](https://github.com/Automattic/wp-calypso/pull/102478/commits/47721ac86c5ebfbad80769119d21f848e1b40a03)